posthog/api/test/test_capture.py

```diff
@@ -53,7 +57,6 @@ def _to_arguments(self, patch_process_event_with_plugins: Any) -> dict:
             "sent_at": sent_at,
         }
 
-    @patch("posthog.models.team.TEAM_CACHE", {})
```

Comment:
since I was updating a lot of tests, felt like this change could be bundled up here.

We got rid of `TEAM_CACHE` a long time ago